sections/featured-product-slider.liquid

```diff
@@ -79,10 +61,9 @@
 
```

Comment:
fix if we have just 2 products on the slider they should be aligned 
<img width="1267" alt="Screenshot 2023-10-09 at 11 00 07" src="https://github.com/youcan-shop/cod-theme-3/assets/108662497/56e076f8-1d0a-49b4-aae1-d95791fe21b8">